{
	"id": "CVE-2020-36225",
	"description": "A flaw was discovered in OpenLDAP before 2.4.57 leading to a double free and slapd crash in the saslAuthzTo processing, resulting in denial of service.",
	"scope": "local",
	"packages": [
		{
			"name": "openldap",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "2.4.57+dfsg-1",
			"repository": [
				{
					"name": "trixie",
					"version": "2.5.18+dfsg-3"
				}
			]
		}
	]
}
